proto/bin_gen.go

```diff
@@ -84,7 +84,7 @@ func systemJson() (*asset, error) {
 		return nil, err
 	}
 
-	info := bindataFileInfo{name: "system.json", size: 30129, mode: os.FileMode(420), modTime: time.Unix(1548273842, 0)}
+	info := bindataFileInfo{name: "system.json", size: 30129, mode: os.FileMode(420), modTime: time.Unix(1548960115, 0)}
```

Comment:
What is this? Was it meant to be part of this PR?

Reply:
I’ll have to revert these changes